changelog/add-7591-missing-columns-export-csv

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: add
+
+Introduce additional columns to the Transaction list UI and CSV export
```

Comment:
I think we should mention the names of the new columns here.

Reply:
Makes sense, updated!